### PR TITLE
[BN] updated comprehension to version 1.1.0

### PR DESCRIPTION
**Pull request by m4hi2**
updated comprehensions translation to match current english document version.

### CODE REVIEW

**Review by wasi0013 — COMMENTED** [review]
Thanks, @m4hi2 bhai, appreciate your effort. There are a few minor issues that we can address now if you want or we can merge them for now and revisit them later when we have finished translating the pending chapters. Let me know your thoughts.

File: lessons/bn/basics/comprehensions.md [hunk 1]
```diff
@@ -68,19 +70,19 @@ iex> for n <- list, times <- 1..n, do: IO.puts "#{n} - #{times}"
...
+ফিল্টারকে আপনি মনে করতে পারেন লিস্ট কম্প্রিহেনশানের গার্ডস্বরপ। ফিল্টারকৃৎ ভ্যালু যদি `false` অথবা `nil` হয় তবে তা রেজাল্ট লিস্ট থেকে বাদ পরে যায়। আমরা এখন একটি রেঞ্জের উপর লুপ করে শুধুমাত্র জোড় সংখ্যা বের করব লিস্ট কম্প্রিহেনশান ও ফিল্টার দিয়ে। এর জন্য আমরা `is_even/1` ফাংশন ব্যবহার করব।
 
 ```elixir
 import Integer
 iex> for x <- 1..10, is_even(x), do: x
 [2, 4, 6, 8, 10]
 ```
 
-জেনারেটরের মত ফিল্টারও একাধিক ব্যবহার করা যায়। যদি আমরা আমাদের রেঞ্জকে বর্ধিত করি এবং শুধুমাত্র সেগুলিকে দেখতে চাই যারা জোড় এবং ৩ দ্বারা বিভাজ্য দুটোই তাহলে নীচের মত করে কোড সাজাব- 
+জেনারেটরের মত ফিল্টারও একাধিক ব্যবহার করা যায়। যদি আমরা আমাদের রেঞ্জকে বর্ধিত করি এবং শুধুমাত্র সেগুলিকে দেখতে চাই যারা জোড় এবং ৩ দ্বারা বিভাজ্য দুটোই তাহলে নীচের মত করে কোড সাজাব-
```
[the remark this answers is not in the document]
Comment (m4hi2) [reply lessons/bn/basics/comprehensions.md:85]: @wasi0013  bhai, yes, "নিচ" is the correct word to use here.

File: lessons/bn/basics/comprehensions.md [hunk 2]
```diff
@@ -68,19 +70,19 @@ iex> for n <- list, times <- 1..n, do: IO.puts "#{n} - #{times}"
...
 ```
 
-লিস্ট কম্প্রিহেনশান হল সিন্ট্যাক্টিক সুগার যা শুধুমাত্র প্রয়োজনভেদে ও যথাস্থানে ব্যবহার করা উচিৎ। 
+লিস্ট কম্প্রিহেনশান হল সিন্ট্যাক্টিক সুগার যা শুধুমাত্র প্রয়োজনভেদে ও যথাস্থানে ব্যবহার করা উচিৎ।
 
-## ফিল্টার 
+## ফিল্টার
 
-ফিল্টারকে আপনি মনে করতে পারেন লিস্ট কম্প্রিহেনশানের গার্ডস্বরপ। ফিল্টারকৃৎ ভ্যালু যদি  `false` অথবা `nil` হয় তবে তা রেজাল্ট লিস্ট থেকে বাদ পরে যায়। আমরা এখন একটি রেঞ্জের উপর লুপ করে শুধুমাত্র জোড় সংখ্যা বের করব লিস্ট কম্প্রিহেনশান ও ফিল্টার দিয়ে। এর জন্য আমরা `is_even/1` ফাংশন ব্যবহার করব। 
+ফিল্টারকে আপনি মনে করতে পারেন লিস্ট কম্প্রিহেনশানের গার্ডস্বরপ। ফিল্টারকৃৎ ভ্যালু যদি `false` অথবা `nil` হয় তবে তা রেজাল্ট লিস্ট থেকে বাদ পরে যায়। আমরা এখন একটি রেঞ্জের উপর লুপ করে শুধুমাত্র জোড় সংখ্যা বের করব লিস্ট কম্প্রিহেনশান ও ফিল্টার দিয়ে। এর জন্য আমরা `is_even/1` ফাংশন ব্যবহার করব।
```
[the remark this answers is not in the document]
Comment (m4hi2) [reply lessons/bn/basics/comprehensions.md]: @wasi0013  bhai, yes.

File: lessons/bn/basics/comprehensions.md [hunk 3]
```diff
@@ -1,25 +1,27 @@
...
   লিস্ট কম্প্রিহেনশান হল সিন্ট্যাক্টিক সুগার যা এলিক্সির এনুমেরাবলে লুপিং করতে সহায়তা করে। এই অধ্যায়ে আমরা দেখব কি করে লিস্ট কম্প্রিহেনশানকে ব্যবহার করতে হয় ইটারেশান ও জেনারেশানের জন্য।
   """
 }
 ---
 
-## Basics
+## বেসিক
 
-`Enum` এবং `Stream` এর মধ্যে লুপিং করতে প্রায়েই কম্প্রিহেনশান ব্যবহৃত হয় এর সিন্ট্যাক্সিয় সংক্ষিপ্ততার জন্য। একটি সরল কম্প্রিহেনশান দেখা যাক-
+`Enum` এবং `Stream` এর মধ্যে লুপিং করতে প্রায়েই কম্প্রিহেনশান ব্যবহৃত হয় কারন এটি লিখিত কোডকে সংক্ষিপ্ত করে। একটি সহজ কম্প্রিহেনশান দেখা যাক:
```
[the remark this answers is not in the document]
Comment (m4hi2) [reply lessons/bn/basics/comprehensions.md]: fixed.

**Review by wasi0013 — APPROVED** [review]

**Comment by wasi0013** [comment]
Thank you @m4hi2 bhai.

**Pull request merged**